plugins/src/core/inApp.ts

```diff
@@ -0,0 +1,23 @@
+export interface InAppInboxItem {
+  id: number;
+  title: string;
+  subtitle: string;
+  availableFrom: string | null; // Date?
+  availableTo: string | null;
+  dismissedAt: string | null;
```

Comment:
availableFrom, availableTo and dismissedAt are being passed as "" from native. nulls are what Flutter does. Can we align native with js? you can add/perhaps even test in android. ill add in swift